apps/condo/domains/common/schema/fields.js

```diff
@@ -5,18 +5,19 @@ const { Json, SignedDecimal } = require('@core/keystone/fields')
 const { ISO_CODES } = require('@condo/domains/common/constants/currencies')
 const { PHONE_WRONG_FORMAT_ERROR, JSON_UNKNOWN_VERSION_ERROR, REQUIRED_NO_VALUE_ERROR, JSON_EXPECT_OBJECT_ERROR } = require('@condo/domains/common/constants/errors')
 const { normalizePhone } = require('@condo/domains/common/utils/phone')
-const { hasValidJsonStructure } = require('@condo/domains/common/utils/validation.utils')
 
 const { ADDRESS_META_FIELD_GRAPHQL_TYPES, ADDRESS_META_SUBFIELDS_QUERY_LIST } = require('@condo/domains/property/schema/fields/AddressMetaField')
 const { UNIT_TYPES, FLAT_UNIT_TYPE } = require('@condo/domains/property/constants/common')
 
+/** @deprecated use dvSenderPlugin! */
```

Comment:
Could also actualize templates?
See `packages/@core.codegen/templates/schema00/domains/common/schema/fields.js.default`

Reply:
ok, I'll keep that in mind for another pull request.